guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -99,20 +137,18 @@ ifdef::foreman-el,katello[]
 # {package-manager} localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-{distribution-major-version}.noarch.rpm
 ----
 endif::[]
+
 ifdef::satellite[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
---enable={RepoRHEL7ServerSatelliteServerProductVersion} \
---enable={RepoRHEL7ServerSatelliteMaintenanceProductVersion} \
 --enable={RepoRHEL7ServerSoftwareCollections} \
---enable={RepoRHEL7ServerAnsible}
+--enable={RepoRHEL7ServerAnsible} \
+--enable={RepoRHEL7ServerSatelliteServerProductVersion} \
+--enable={RepoRHEL7ServerSatelliteMaintenanceProductVersion}
 ----
 endif::[]
-ifdef::foreman-el,katello[]
-include::proc_configuring-repositories-el.adoc[]
-endif::[]
```

Comment:
why are you removing this? it is still relevant on non-Satellite.

Reply:
It was extra and caused checks failure. Foreman and Katello will not face any issues as it is covered for both.